glean-core/rlb/tests/schema.rs

```diff
@@ -88,8 +89,62 @@ fn validate_against_schema() {
     };
     let _ = new_glean(Some(cfg));
 
-    // Define a new ping and submit it.
     const PING_NAME: &str = "test-ping";
+
+    // Test each of the metric types, just for basic smoke testing against the
+    // schema
+
+    // TODO: 1695762 Test all of the metric types against the schema from Rust
+
+    let rate_metric: RateMetric = RateMetric::new(CommonMetricData {
+        name: "rate".into(),
+        category: "test".into(),
+        send_in_pings: vec![PING_NAME.into()],
+        ..Default::default()
+    });
+    rate_metric.add_to_numerator(1);
+    rate_metric.add_to_denominator(1);
+
+    let numerator_metric1: NumeratorMetric = NumeratorMetric::new(CommonMetricData {
+        name: "num1".into(),
+        category: "test".into(),
+        send_in_pings: vec![PING_NAME.into()],
+        ..Default::default()
+    });
+    let numerator_metric2: NumeratorMetric = NumeratorMetric::new(CommonMetricData {
+        name: "num2".into(),
+        category: "test".into(),
+        send_in_pings: vec![PING_NAME.into()],
+        ..Default::default()
+    });
+    let denominator_metric: DenominatorMetric = DenominatorMetric::new(
+        CommonMetricData {
+            name: "den".into(),
+            category: "test".into(),
+            send_in_pings: vec![PING_NAME.into()],
+            ..Default::default()
+        },
+        vec![
+            CommonMetricData {
+                name: "num1".into(),
+                category: "test".into(),
+                send_in_pings: vec![PING_NAME.into()],
+                ..Default::default()
+            },
+            CommonMetricData {
+                name: "num2".into(),
+                category: "test".into(),
+                send_in_pings: vec![PING_NAME.into()],
+                ..Default::default()
+            },
+        ],
+    );
```

Comment:
@chutten pointed out in Matrix that maybe we should document how to make these in the developer docs.  First, to confirm that this is correct, though...

Reply:
This looks correct to me.